Source/Panels.h

```diff
@@ -771,6 +772,16 @@ std::vector<int> squarePanels = {
     0x09E85, // Tunnels Town Shortcut
 };
 
+std::set<int> doubleModeBannedSquarePanels = {
```

Comment:
```suggestion
std::vector<int> doubleModeBannedSquarePanels = {
```
There is no noticable perf benefit to set over vector when the data set is small. Let's use vector for consistency.

Reply:
So, I just tried this and apparently vector doesn't have a find method? We could either use algorithm's std::find, or switch back to set, whichever you prefer.

Reply:
std::find is ok by me.